travis/shared_configs/python-tester-conda.yml

```diff
@@ -4,9 +4,9 @@ jobs:
   include:
     - &testpythonconda
       stage: test
-      name: "Python 3.6"
+      name: "Python 3.7"
       env:
-        - PYTHON_VERSION: 3.6
+        - PYTHON_VERSION: 3.7.3
```

Comment:
Seems old? Not 3.7.10?

Reply:
That's just the version that was existing at https://github.com/pcdshub/pcds-ci-helpers/blob/14f695d5c796a55804796ae668d1d181c8fc10f4/travis/shared_configs/python-tester-conda.yml#L54

I didn't think very hard about it and there isn't much justification in the commit history or in past PRs, I'll just drop the minor pin entirely as is done in other versions

Reply:
Sounds good